src/lib/methods/batch/batch-write.ts

```diff
@@ -0,0 +1,102 @@
+import { Executable } from '../executable';
+import { DynamoDB } from '../../dynamodb';
+import { BatchItem } from './batch-item';
+import { BatchMethod } from './batch-method';
+import { BatchWriteItemInput, BatchWriteItemOutput } from 'aws-sdk/clients/dynamodb';
+
+export class BatchWrite extends BatchMethod implements Executable {
+
+	constructor(
+		dynamodb: DynamoDB,
+		private items: BatchItem[],
+		private msDelay = 1,
+		private retryCount = 10) {
+		super(dynamodb);
+	}
+	/**
+	 * Initialize the `InsertItem` object.
+	 */
+	initialize(): BatchWrite {
+		const request = {
+			RequestItems: {}
+		};
+		this.items.forEach((item) => {
+			const table = request.RequestItems[item.table];
+			if (!table) {
+				request.RequestItems[item.table] = [item.value];
+			} else {
+				(request.RequestItems[item.table] as any[]).push(item.value);
+			}
+		});
```

Comment:
Use a regular `for-of`. I don't like `forEach` 🤷 :).

```suggestion
		this.items.forEach((item) => {
			const table = request.RequestItems[item.table];
			if (!table) {
				request.RequestItems[item.table] = [item.value];
			} else {
				(request.RequestItems[item.table] as any[]).push(item.value);
			}
		});
```